modules/core/src/main/java/webindex/core/IndexClient.java

```diff
@@ -231,4 +249,79 @@ public Links getLinks(String rawUrl, String linkType, String next, int pageNum)
     }
     return links;
   }
+
+  public static Collection<Mutation> genDomainMutations(DomainUpdate update, long seq) {
+    Map<RowColumn, Bytes> oldData = genDomainData(update.getDomain(), update.getOldPageCount());
+    Map<RowColumn, Bytes> newData = genDomainData(update.getDomain(), update.getNewPageCount());
+    return AccumuloExportQueue.generateMutations(seq, oldData, newData);
```

Comment:
I did not realize this while looking at the recipes PR, but looking at this PR made me realize that generateMutations is a really generic name.